src/config/dynamic-theme-fixes.config

```diff
@@ -14524,6 +14524,30 @@ CSS
 
 ================================
 
+smash.gg
+
+CSS
+:root {
+    --darkreader-bg--shadow--card: 0 2px 4px rgba(0,0,0,0.4) !important;
+}
+.ModalContext__close {
+    border-bottom: none !important;
+}
+div[class^="controller"] {
+    background: none !important;
+}
+div[class^="BracketContainer"] > svg > path {
+    stroke: #373c3e !important;
```

Comment:
Can this be converted into the `#373c3e` -> `${original color here}`?

Reply:
do you mean that i should just put the hex into the ${...} or should i find in the source code of smash.gg if there is a css color variable for it.

(btw there is no css color name for this exact hex)

Reply:
You should put the original hex from smash.gg's theme into `${HERE}`, such that darkreader will modify it correctly according the user's settings.

Reply:
"the original hex" => so basically the inverted hex of what is in here? what about custom colors that i chose here, e.g.:

rgb(177 157 157 / 10%)

Reply:
also, i have another problem: smash.gg updates its page automatically (spa) and when said elements are being created, the darkreader styles don't apply.

Reply:
> so basically the inverted hex of what is in here?

Yup, but they can also be in RGB(A) format if needed, dark reader can automatically convert them.



> what about custom colors that i chose here, e.g.:
> 
> rgb(177 157 157 / 10%)

Avoid them at all time, you can insert `${#B19D9D}` and see what dark reader's result it and put that in this PR(as modification function that dark reader uses, is as well the inverse function of itself).

Reply:
> also, i have another problem: smash.gg updates its page automatically (spa) and when said elements are being created, the darkreader styles don't apply.

I don't think that's a problem, many sites are these days and Dark Reader watches for updates just fine.

Reply:
> > also, i have another problem: smash.gg updates its page automatically (spa) and when said elements are being created, the darkreader styles don't apply.
> 
> I don't think that's a problem, many sites are these days and Dark Reader watches for updates just fine.

I can confirm that it is an issue because reapplying the styles when the page is loaded already makes it work as it should.

Reply:
Please open up a new issue about it then, it's _quite_ unrelated to this PR.

Reply:
alright. i will do the color fixes asap